mf4_csv/main.py

```diff
@@ -0,0 +1,52 @@
+
+import os 
+from asammdf import MDF, Signal
+from os import walk
+import re
+import glob
+
+
+output_dir = "./output_in_csv"
+
+def prepare_dir(mf4_file):
+    my_str = str(mf4_file)
+    my_str = my_str.lower().strip(".mf4")
+    my_str = my_str.replace(".MF4", ".csv")
```

Comment:
Will this code actually replace the .MF4 extension? If it is being stripped in the previous statement with `my_str.lower().strip("mf4")` the string should no longer contain any file extension.

Reply:
The approach is to create a csv file with the same name as the original mf4 file in the same directory. The strip is to remove the extension and use the other part of the name and directory.